.github/workflows/docker.yml

```diff
@@ -10,6 +10,7 @@ name: Build / Release Docker Images
 jobs:
```

Comment:
I just realized why my branches never ended up running the docker release action: I have always used `/` in my branch names:
```yaml
on:
  push:
    branches:
    - '*' // this excludes branches with '/' in them
```

Can you update that to ** please?

Reply:
Sure! I also want to wait for my other merge to main to actually finish building before I merge this.